src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## Unreleased
+
+* BUGFIX: Fix issue `merge` command generates SARIF with multiple runs [#2488](https://github.com/microsoft/sarif-sdk/pull/2488)
```

Comment:
>Fix [](http://example.com/codeflow?start=10&length=3)

This description does not clearly describe the issue and the fix. @marmegh or @easyrhinomsft can you please help Yong with this?

Reply:
Updated pls review. cc @VCOLAWTON

Reply:
So, let's look at what we're fixing here.

The problem: the merge command was generating a sarif with a run per rule instead of a run per tool.

The description here does not capture that. We need to tell the customers what behavior they may have been experiencing that is now changed. Specific first, concise second.

"Fixed `merge` command producing runs per rule to the expected runs per tool."

Reply:
Thanks for the suggestions. Updated both, pls review.